src/window.rs

```diff
@@ -116,6 +116,26 @@ pub struct WindowAttributes {
     /// The default is `None`.
     pub max_inner_size: Option<Size>,
 
+    /// The desired position of the window. If this is `None`, some platform-specific position
+    /// will be chosen.
+    ///
+    /// The default is `None`.
+    ///
+    /// ## Platform-specific
+    ///
```

Comment:
The Platform-specific list should be formatted as a list, that is each item beggining with a `-` like this:
```
- **macOS**: ...
- **Window**: ...
```

Also under Platform-specific, you should describe the kind of position this means on all of the platforms. For example
```
- **X11**: The outer position.
- **Others**: This is ignored.
```